.github/workflows/tests.yml

```diff
@@ -45,7 +45,11 @@ jobs:
       - name: Select Xcode 11.2
         run: sudo xcode-select -s /Applications/Xcode_11.2.app
       - name: Generate Xcode projects and compare against existing record
-        run: (bazelisk query 'kind(xcodeproj, tests/macos/xcodeproj/...)' | xargs -n 1 bazelisk run) && git diff --exit-code tests/macos/xcodeproj
+        run: |
```

Comment:
Nice, more readable! Can we count on all the lines being run in the same shell? Otherwise the `set -euo pipefail` might not be taking effect...

Reply:
The whole `run` string gets executed at once by the shell